boards/m5stack/m5stack_cores3/m5stack_cores3_procpu_defconfig

```diff
@@ -5,3 +5,5 @@ CONFIG_CONSOLE=y
 CONFIG_SERIAL=y
 CONFIG_UART_CONSOLE=y
 CONFIG_GPIO=y
+CONFIG_I2C=y
```

Comment:
I think `CONFIG_I2C=y` isn't necessary as it is automatically added when enabled in DTS file. Can you confirm whether you need this here?

Reply:
Thank you for your comment.

I found the correct way to fix it.
The device can be disabled in the MCUBoot settings, so I changed it to deal with the problem that way.